README.md

```diff
@@ -182,6 +182,12 @@ The BTCPay Server Project is proudly supported by these entities through the [BT
           <span>ACINQ</span>
         </a>
       </td>
+       <td align="center" valign="middle">
+        <a href="https://lunanode.com" target="_blank">
```

Comment:
lowercase

Reply:
it should be lowercase it's a goddamn link